src/cheri_insts.sail

```diff
@@ -783,7 +783,7 @@ function clause execute (CClear(q, m)) = {
       if q_u == 0 & i == 0 then
         DDC = null_cap
       else
-        C(8 * q_u + i) = null_cap;
+        wC (false, 8 * q_u + i, null_cap);
```

Comment:
This is the bit that will show in the ISA doc, and the no-context `false` may be confusing. I wonder if there's a clearer way, for example wrapping wc(false, ....) in a function. Not sure what to call that function though, since RVFI isn't really visible from the instruction-level description.